.github/workflows/release.yaml

```diff
@@ -10,16 +10,34 @@ on:
     workflow_dispatch:
 
 jobs:
-    publish:
+    publish-cli:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
-            - uses: actions/setup-node@v3
+            - uses: actions/setup-node@v4
               with:
-                  node-version: "18.x"
-                  registry-url: "https://registry.npmjs.org"
+                  node-version: 20
+                  registry-url: https://registry.npmjs.org
             - run: npm ci
             - run: npm run build
             - run: npm publish
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+    publish-resources-lib:
```

Comment:
I believe this should be run only if there are changes in `src/projectConfiguration/ts/resources`.

Right now it seems to me that this is run for every cli release and these releases are not necessarily tightly coupled.